src/utils/number.js

```diff
@@ -17,7 +17,6 @@ export function isInteger (value) {
   return isFinite(value)
     ? (value === Math.round(value))
     : false
-  // Note: we use ==, not ===, as we can have Booleans as well
```

Comment:
False comment

Reply:
Good find, feel free to remove the comment